lambdas/service/app.py

```diff
@@ -1524,6 +1528,26 @@ def handle_manifest_generation_request() -> Tuple[int, Manifest]:
                     catalog=IndexName.validate_catalog_name,
                     filters=str,
                     token=str)
+
+    def amend_manifest_location(manifest: Manifest, token: str = None):
```

Comment:
If `None` is an acceptable value for the `token` parameter, the type annotation should reflect this.

```suggestion
    def amend_manifest_location(manifest: Manifest, token: Optional[str] = None):
```